pages/_app.js

```diff
@@ -1,7 +1,24 @@
 import '../styles/globals.css'
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+      <div>
+        <link
+          rel="stylesheet"
+          href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        />
+        <link
+          href="https://fonts.googleapis.com/css?family=Oswald"
+          rel="stylesheet"
+        />
+        <link
+          rel="stylesheet"
+          href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css"
```

Comment:
Can you extract out the grid from bootstrap. As we are not using other modules.

Reply:
Okay